### PR TITLE
CLDR-17590 Update Locales.txt for v46

### PR DESCRIPTION
**Pull request by AEApple**
CLDR-17590

Update Locales.txt for v46 submission

- [x] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Review by srl295 — APPROVED** [review]

**Review by AydaTolaymat — APPROVED** [review]
looks good, thank you.

**Comment by AEApple** [comment]
Thanks! I'm going to wait for Mark to review the other language set before merging.

**Review by macchiati — APPROVED** [review]
Looks great.

**Pull request merged**
